src/component/Main/Forms/AddCar.js

```diff
@@ -0,0 +1,133 @@
+/* eslint linebreak-style: ["error", "windows"] */
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import NavBar from '../NavBar';
+import Mobilemenu from '../Mobilemenu';
+import NavigationPanel from '../NavigationPanel';
+import { addLatestCarModel } from '../../../redux/carSlice';
+import '../../../style/addcar.css';
+import lambergini from '../../../assets/9296454.gif';
+
+export default function AddCar() {

```

Comment:
- [ ] The functionality for adding cars is in the code, however, the button "Add car" does not work:


![image](https://github.com/Log-benjamin/Wheels-4-Rent-frontend/assets/57421823/04fd650a-ec13-4c61-88fa-5ac3ee666d57)

This is a core requirement for teams of 2.  Kindly fix the button so your users are able to add cars. 👍

Reply:
@Meltrust Thank you for taking the time to reivew our project and we will make the changes per your suggestions.
for this specific feature we wlready have implemented it. you just have to log in ussing the default ADMIN CREDENTIAL                 username: admin ,     password: 123456789
I have mentiond this in the PR Description 

![image](https://github.com/Log-benjamin/Wheels-4-Rent-frontend/assets/97245616/e26dce75-ad64-4400-a6d6-52cfc2191ad0)

![image](https://github.com/Log-benjamin/Wheels-4-Rent-frontend/assets/97245616/f4e58f0d-eded-4d26-905b-4001470ad934)